spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -43,4 +43,8 @@ private SparkSQLProperties() {
   // Controls whether to check the order of fields during writes
   public static final String CHECK_ORDERING = "spark.sql.iceberg.check-ordering";
   public static final boolean CHECK_ORDERING_DEFAULT = true;
+
+  // Control whether to parallel delete files when abort
+  public static final String DELETE_FILES_PARALLEL_WHEN_ABORT = "spark.sql.iceberg.delete-files-parallel-when-abort";
```

Comment:
The name "delete-files-parallel-when-abort" is very long. How about "parallel-abort-enabled"?

Does this even need to be an option? Why would anyone turn it off?

Reply:
@rdblue I remove the option. Previously I want to have the ability to keep the behavior as before.